### DISCUSSION TITLE
How to change the color for a MIXED-CHART?

### DISCUSSION THREAD

**Discussion by tsator** [apache/superset#28513] [Q&A / Help]
Hi!, i'm trying change the color for a superset mixed-chat but i cant do it. I usually change the color using JSON by modifying the labels_colors but for this type of chart it does not work. I tried solutions that were proposed in other discussions such as adding the sufix (1) after the label name but it doesnt work either. Is there any solution for this?

<img width="113" alt="image" src="https://github.com/apache/superset/assets/135351271/14d3033e-2553-4301-a1d0-75df86cc1e5b">

![image](https://github.com/apache/superset/assets/135351271/58302e85-1adc-47c6-b698-777665f9760a)

Im currently using superset 4.0

**Comment by ari-jane** [comment]
I had the same issue. The label colors worked for all lines that were created by Query A, but not for the ones of Query B.
However, when I used the suffix (1), so in my case `"total average (1)": "#E1292B",` instead of `"total average": "#E1292B",` it worked.
Is your "Cantidad episodios" in Query A or B? If in Query A, maybe try the suffix (0). Also you might check if just using "red" to select the color actually works or if you have to use hex color (you can try out with the hex color above, its a bright red).

**Comment by tsator** [comment]
This is mi JSON,

  "label_colors": {
    "Cantidad episodios": "#C3310E",     (I also tried using "Cantidad episodios (0)")
    "Estancia media (1)": "#C3310E"}

And the labels;
- "Cantidad episodios" is in Query A
- "Estancia media" is in Query B

However, the colors are not apllied to any variable


![image](https://github.com/apache/superset/assets/135351271/958328dd-67b5-4cf3-8112-6f210cc8c22a)

![image](https://github.com/apache/superset/assets/135351271/60d106bc-6049-4947-8a74-3452b4057cae)

<img width="67" alt="image" src="https://github.com/apache/superset/assets/135351271/368f9ab5-87ff-4246-9620-19ec7a692d20">
